### PR TITLE
Removed duplicate Hainan region record

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/463
- Added `edtf:cessation` and `mz:is_current` values to "incorrect" Hainan record.
- Superseded "incorrect" Hainan record into second Hainan record, vice versa.
- Brought over `ne` fields.
- Exportified.

### CODE REVIEW

**Comment by nvkelso** [comment]
Overall looks good. But each of the 25 descendants need their hierarchy updated to only refer to the final feature, not both earlier features. For instance: https://whosonfirst.mapzen.com/spelunker/id/890513893/#18/18.83778/110.28750.

**Comment by stepps00** [comment]
Commit b21cb8c updated the hierarchies (and exportified) each of the 25 descendants of Hainan.

File: data/102/027/079/102027079.geojson [hunk 1]
```diff
@@ -64,20 +63,14 @@
...
         }
     ],
     "wof:id":102027079,
     "wof:lang":[
         "chi"
     ],
-    "wof:lastmodified":1470254941,
+    "wof:lastmodified":1474404906,
     "wof:name":"Wenchang Shi",
     "wof:parent_id":-1,
```
Comment (nvkelso) [inline data/102/027/079/102027079.geojson]: Should the `wof:parent_id` be `85669759` now?

File: data/890/511/875/890511875.geojson [hunk 2]
```diff
@@ -18,11 +18,11 @@
     "qs:photos_9r":"0",
     "qs:photos_sr":"0",
     "qs:pop_sr":"8",
+    "src:geom":"unknown",
```
Comment (nvkelso) [inline data/890/511/875/890511875.geojson]: This was probably `quattroshapes` (not unknown).

File: data/102/027/079/102027079.geojson (hunk 1 above)
Comment (nvkelso) [inline data/102/027/079/102027079.geojson]: This is happening on other of the files, too.

File: data/890/511/875/890511875.geojson (hunk 2 above)
Comment (nvkelso) [inline data/890/511/875/890511875.geojson]: This is happening on other of the files, too.

**Comment by stepps00** [comment]
@nvkelso - All records should now have correct `parent_id` property values and `src:geom` property values. The files updated in the most recent commit also had their `lastmodified` property updated.

**Comment by nvkelso** [comment]
LGTM, but merge conflicts have arisen.

**Comment by stepps00** [comment]
Most recent [commit](819345f) fixed merge conflicts.

**Pull request merged**